src/katsdpk8spoc/workflow_controller.py

```diff
@@ -165,7 +178,9 @@ def __init__(self, number: int):
 
 
 class ProductControllerWorkflow:
-    def __init__(self, namespace: int, worker_count: int = 10, ttl: int = 600):
+    def __init__(
+            self, namespace: int, config: dict, worker_count:
+            int = 10, ttl: int = 600):
```

Comment:
The worker count should be specified/calculated. 
The ttl should maybe also have a config default?

Reply:
TTL should be in config. 
For us, in the PoC I think we should not use a number but have a method or a class that returns a list of batch Jobs and their settings. Let's assume a class for now. 
This class will internally guess* some number and return a bunch of simple Jobs, but in reality, this requires a bit more thinking, the tickets for later in the PI. Such a class will need to static config (our config) and the config passed in from CAM with product controller. For the PoC we just spin up a batch job for every receptor, but if we hide this nicely in its own class then we can just say "add you logic here".

Reply:
I added a fake worker count method and added the TTL to the config.